sdk/appcenter-analytics/src/test/java/com/microsoft/appcenter/analytics/AnalyticsTest.java

```diff
@@ -190,10 +200,82 @@ public boolean matches(Object item) {
     public void trackEventFromApp() {
```

Comment:
I would prefer for this test to be split up:
- send without properties
- send event with empty properties
- send event with non-map properties
- prepare and validate typed properties

Right now if this test fails it's very unclear what scenario isn't working.

Reply:
I like this one!